app/views/searches/index.html.erb

```diff
@@ -21,7 +21,10 @@
       //   that.send('search');
       // }
     });
-
-    new CBPGridGallery(document.getElementById('grid-gallery'));
+    // Init the image gallery
```

Comment:
This could be placed in app/assets/javascripts/searches.js.coffee, if we require the tree then it should autoload, alternatively we can load just the specific controller scripts via 
```<%= javascript_include_tag params[:controller] %>```
 but we need to require tree false.